releases/release-1.25/release-notes/maps/pr-111441-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 111441
+releasenote:
+  text: Respect PodTopologySpread after rolling upgrades.
```

Comment:
```suggestion
  text: Respected PodTopologySpread after rolling upgrades.
```

Reply:
Changed this a bit to make the message clearer.